app/jinja_filters.py

```diff
@@ -212,12 +237,12 @@ def should_wrap_with_fieldset(question):
 
 
 @blueprint.app_context_processor
-def should_wrap_with_fieldset_processor():
+def should_wrap_with_fieldset_processor() -> dict[str, Callable]:
     return {"should_wrap_with_fieldset": should_wrap_with_fieldset}
 
 
 @blueprint.app_template_filter()
-def get_width_for_number(answer):
+def get_width_for_number(answer: AnswerType) -> int:
```

Comment:
This method not always has a return (if width is > allowable_width) so needs to be `Optional`.